<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogImpostazione</class>
 <widget class="QDialog" name="DialogImpostazione">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>325</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>325</width>
    <height>430</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>325</width>
    <height>430</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>325</width>
    <height>430</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Monospace</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="EditDialogHostname">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>215</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelHost">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>218</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Hostname</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btSave">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>396</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditDialogPwd">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>301</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>304</y>
     <width>65</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>65</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>65</width>
     <height>20</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>65</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditDialogDatabase">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>242</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelDatabase">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>245</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Database</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditDialogUsername">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>272</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelUsername">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>275</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Utente</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSmtpPassword">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>123</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>SMTP Passwd</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditDialogSmtpUser">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>90</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelSmtpServer">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>37</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>SMTP Server</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditDialogSmtpPassword">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>120</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelSmtpUser">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>93</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>SMTP User</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditDialogSmtpServer">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>34</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>150</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelImpostazioneMail">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Impostazione Mail</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelImpostazioneDB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>175</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Impostazione Database</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelImpostazioneSave">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>399</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Salva</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSmtpSender">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>66</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>SMTP Sender</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditDialogSmtpSender">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>63</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelImpostazioneDaemonPort">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Impostazione Servizio</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelDaemonPort">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>363</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxDaemonPort">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>360</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>9999</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>EditDialogSmtpServer</tabstop>
  <tabstop>EditDialogSmtpSender</tabstop>
  <tabstop>EditDialogSmtpUser</tabstop>
  <tabstop>EditDialogSmtpPassword</tabstop>
  <tabstop>EditDialogHostname</tabstop>
  <tabstop>EditDialogDatabase</tabstop>
  <tabstop>EditDialogUsername</tabstop>
  <tabstop>EditDialogPwd</tabstop>
  <tabstop>btSave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
